website/sidebars.js

```diff
@@ -5,7 +5,7 @@ function FBInternalOnly(elements) {
 }
 
 module.exports = {
-    Docs: {
+    docs: {
```

Comment:
The docs plugin expect a sidebar with name 'docs', not 'Docs', this is why your site breaks on upgrade. We should make this more robust in the future.

Reply:
> Note, it's actually upgrading the dependency from 1.1.0 to 1.1.1 that seems to break the highlighting, so I forced the former using:
> 
> ```
>   "resolutions": {
>     "prism-react-renderer": "1.1.0"
>   },
> ```
> 
> You likely upgraded without knowing as it's a minor semver range

Seems likely, thanks again. I will file an issues once I confirm this locally.

Reply:
Ah, saw that you already did. Thanks!